charts/values.yaml

```diff
@@ -350,13 +331,13 @@ haproxy:
   ## You need to put the PEM-formatted SSL certificate into a secret and provide the secret name in the secretName field.
```

Comment:
"Put the PEM-formatted SSL certificate into a secret and provide the secret name in the secretName field."